app_base.htm

```diff
@@ -0,0 +1,70 @@
+## APP BASE CONTAINS THE NAV BAR
+
+<%inherit file="/homepage/templates/base.htm" />
+import
```

Comment:
This "import" looks unnecessary... I'd recommend removing it.

Reply:
thx for catching that